downstream.patch

```diff
@@ -0,0 +1,44 @@
+diff --git a/apb/apb.yml b/apb/apb.yml
+index d7610ca9..73833fc8 100644
+--- a/apb/apb.yml
++++ b/apb/apb.yml
+@@ -14,7 +14,7 @@ metadata:
+     WARNING!!! Use of this APB through the service-catalog is
+     not supported.
+   dependencies:
+-    - 'docker.io/ansibleplaybookbundle/origin-ansible-service-broker'
++    - 'registry.redhat.io/openshift3/ose-ansible-service-broker'
+   displayName: Automation Broker (APB)
+   providerDisplayName: "Red Hat, Inc."
+ plans:
+diff --git a/apb/defaults/main.yml b/apb/defaults/main.yml
+index 0ff227b3..7cf17b66 100644
+--- a/apb/defaults/main.yml
++++ b/apb/defaults/main.yml
+@@ -12,7 +12,7 @@ wait_for_broker: false # By default, we do not wait for the broker to be availab
+ 
+ # The image to use for the broker
+ broker_image_tag: latest
```

Comment:
I think the desire is to have this match the release (v3.11, for instance) rather than latest so as additional releases come along they get a broker in line with the correct release.